magento/magento2
# Fix custom_options Import doesn't correctly handle '=' and multi-value separators in fields #17597

### Description (*)
The import function in System -> Import doesn't correctly handle custom_options with the multi-value separator field in a value as well as the '='. A simplistic patch attached. Also attached sample data.

### Fixed Issues (if relevant)
1. magento/magento2#17597: custom_options Import doesn't correctly handle '=' and multi-value separators in fields

### Manual testing scenarios (*)
Steps to reproduce
1. Create a Product with options containing '=' in Title.
2. Export products.
3. Import product CSV, created in Step 2, selecting 'Add/Update' Import Behavior.
4. Observe Product options.
##### Expected result
Options are the same as before export.
##### Actual result
Options are modified - '=' and all text after it is absent.

##### Examples
eg. name=EN = #10 Envelope

Expected:
EN = #10 Envelope

Actual:
EN

eg. option_title=10,000

Expected:
10,000

Actual:
10

### Contribution checklist (*)
